framework/util/file_path.cpp

```diff
@@ -435,6 +435,28 @@ void GetApplicationInfo(FileInfo& file_info)
     {
         GFXRECON_LOG_WARNING("Failed to retrieve the application executable name");
     }
+#elif defined(__ANDROID__) || defined(__linux__)
+    std::ifstream cmdlineFile("/proc/self/cmdline");
+
+    if (cmdlineFile.is_open())
+    {
+        std::string line;
+        std::getline(cmdlineFile, line);
```

Comment:
Can you do one better here and use `istream::getline`?  I think that would allow you to skip the copy.  Something like `        cmdlineFile.getline(file_info.AppName, sizeof(file_info.AppName), '\0');`?

Reply:
Please review the current state. if you think that we should still change it, I can.

Reply:
Done